docs/site/_includes/menu.njk

```diff
@@ -64,7 +64,6 @@
             <div class="expanded-menu-dropdown">
               <a class="expanded-menu-dropdown-link js-event-hm-menu" href="/style/content/" tabindex="-1">Content style guide</a>              
               <a class="expanded-menu-dropdown-link js-event-hm-menu" href="/style/design/" tabindex="-1">Design style guide</a>
-              <a class="expanded-menu-dropdown-link js-event-hm-menu" href="/style/engineering/" tabindex="-1">Engineering style guide</a>
```

Comment:
Do we need tabindex here?

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

Reply:
I think so. The `tabindex=-1` is removed by JS when the dropdown menu is expanded. In this default state, it prevents keyboard navigation from focusing on links that have not yet been revealed.